libs/platform/src/lib/components/form/platform-file-uploader/platform-file-uploader.component.ts

```diff
@@ -132,6 +133,11 @@ export class PlatformFileUploaderComponent extends BaseInput {
         super(_cd, ngControl, ngForm, formField, formControl);
     }
 
+    /** @hidden */
+    ngOnInit(): void {
```

Comment:
Do we need it here?

Reply:
@N1XUS 
it is required to connect form-field label with form-field input.
steps:
1. https://github.com/SAP/fundamental-ngx/blob/6ab69d630c1bbfc00e77cee1e708f1d3358c63a0/libs/platform/src/lib/components/form/base.input.ts#L196
2.https://github.com/SAP/fundamental-ngx/blob/6ab69d630c1bbfc00e77cee1e708f1d3358c63a0/libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts#L355

Reply:
@N1XUS, please see my reply.
if all looks fine, please approve it.